regression/tests/studio/test_studio_grading.py

```diff
@@ -50,3 +62,41 @@ def test_grade_range(self):
         self.browser.refresh()
         self.assertEquals(
             self.grading_page.letter_grade('.letter-grade'), 'Pass')
+
+    def test_assignment_types(self):
+        """
+        Verifies that user can add/delete assignment types
+        """
+        self.grading_page.click_new_assignment_type()
+        self.grading_page.fill_assignment_type_fields(
```

Comment:
Because there are so many fields, I think you should be explicit about this method call. For example, 

```python
self.grading_page.fill_assignment_type_fields(
  name='Final',
  abbreviation='Finale'
  ...etc...
)
```